relayer/relayer/src/trader/mod.rs

```diff
@@ -0,0 +1,303 @@
+use std::{
+    fmt::Debug,
+    str::FromStr,
+    sync::Arc,
+    time::{Duration, SystemTime, UNIX_EPOCH},
+};
+
+use aleph_client::{pallets::system::SystemApi, AccountId, AsConnection, SignedConnectionApi};
+use ethers::providers::Middleware;
+use log::{debug, error, info, warn};
+use thiserror::Error;
+use tokio::{select, sync::broadcast, time::sleep};
+
+use crate::{
+    config::Config,
+    connections::{azero::AzeroConnectionWithSigner, eth::SignedEthConnection},
+    contracts::{AzeroContractError, AzeroEtherInstance, MostInstance, RouterInstance},
+    helpers::left_pad,
+    CircuitBreakerEvent,
+};
+
+// trader component will sell the surplus
+pub const ONE_AZERO: u128 = 1_000_000_000_000;
+pub const ONE_ETHER: u128 = 1_000_000_000_000_000_000;
+
+pub const ETH_TO_AZERO_RELAYING_BUFFER: u128 = 100 * ONE_AZERO;
```

Comment:
Seems to me this is the threshold, so perhaps that comment was meant to be here?